packages/antd/package.json

```diff
@@ -76,38 +76,17 @@
     "rc-picker": "2.7.6"
   },
   "devDependencies": {
-    "@ant-design/icons": "^5.6.1",
-    "@babel/cli": "^7.23.9",
-    "@babel/core": "^7.23.9",
-    "@babel/eslint-parser": "^7.23.10",
-    "@babel/plugin-proposal-class-properties": "^7.18.6",
-    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
-    "@babel/preset-env": "^7.23.9",
-    "@babel/preset-react": "^7.23.3",
-    "@babel/preset-typescript": "^7.23.3",
+    "@ant-design/icons": "6.0.0",
```

Comment:
Is the intent behind `6.0.0` instead of `^6.0.0` to stabilize things like snapshots and typechecks so they only change when we explicitly upgrade?

Reply:
Ah, no I just used an alias that does things exactly since that is the way we do it at work. I can add back the `^`